Issue #59 - New Footer

EChesters opened the pull request:
- Added footer in every HTML page
- Added bootstrap and font-awesome
- Added custom CSS for footer

Footer screenshot included in original issue description.

EChesters committed:
Issue #59
+ Added footer in every HTML page
+ Added bootstrap and font-awesome
+ Added custom CSS for footer

lili2311 commented:
is there a screenshot of finished pages?

lili2311 commented on src/index.html:
```diff
@@ -23,6 +23,8 @@
     <meta name="twitter:description" content="Women Hack For Non-Profits | Home" />
     <meta name="twitter:image" content="img/whfnpLogoSocialMediaIcon.png" />
     <!--Styles-->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
```
- i would improve this by using [bootsrap sass](https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/_bootstrap.scss) and only importing what is needed into the current sass example [here](https://github.com/womenhackfornonprofits/fashion-revolution-django/tree/master/static-src/styles)

The site is not performing great as it is this is extra resources to load.

EChesters committed:
Added npm/grunt dependency for bootstrap-sass
+ amended head section in all HTML files
+ left in comments bootstrap components incase needed later

EChesters commented:
Another screenshot of the footer...
<img width="1680" alt="screen shot 2016-04-19 at 23 10 05" src="https://cloud.githubusercontent.com/assets/2605703/14657238/83f7a7b0-0684-11e6-853d-fab818884389.png">

lili2311 commented:
nice! is the font the same in the footer it looks completely different? Might have to be set specifically, otherwise 👍

EChesters committed:
Changed font for footer

lili2311 commented on src/volunteers.html:
```diff
@@ -188,13 +189,50 @@ <h2 class="section-heading">Newsletter</h2>
       </div>
     </footer>
-      <script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
```
do we need this? what is using this?

EChesters replied:
This is used by Bootstrap.

lili2311 replied:
bootstrap is not depended on jquery, if we re not using any of the javascript plugins and only using css/sass it is not needed.

EChesters replied:
Bootstrap does depend on JQuery. Sometimes it can break things like layouts as well but I'm guessing for this footer it's not using it.

EChesters committed:
Removed unused JS for Bootstrap
+ This had only been put on the volunteers page
